Services/Init/classes/class.ilInitialisation.php

```diff
@@ -1359,6 +1359,12 @@ protected static function initUIFramework(\ILIAS\DI\Container $c) {
 					( $c["ui.resource_registry"]
 					, new ILIAS\UI\Implementation\Render\FSLoader
 						( new ILIAS\UI\Implementation\Render\DefaultRendererFactory
+							($c["ui.factory"]
+							, $c["ui.template_factory"]
+							, $c["lng"]
+							, $c["ui.javascript_binding"]
+							),
+						  new ILIAS\UI\Implementation\Component\Glyph\GlyphRendererFactory
```

Comment:
Why is this necessary?

Reply:
Because Glyphs need to be rendered without an `<a>`-Tag when they are contained in a Graphical Button.